### PR TITLE
* Add lcx installation method

### PR DESCRIPTION
**Pull request by bmsleight**

### CODE REVIEW

File: docs/examples/lxc/lxc-install.sh [hunk 1]
```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env bash
+
+# Bash script to install paperless in lxc containter
+# paperless.lan
+#
+# Will set-up paperless, apache2 and proftpd
+#
+# lxc launch ubuntu: paperless
+# lxc exec paperless -- sh -c "wget https://raw.githubusercontent.com/danielquinn/paperless/master/docs/examples/lxc/lxc-install.sh && /bin/bash lxc-install.sh"
```
Comment (MasterofJOKers) [inline docs/examples/lxc/lxc-install.sh]: Please change the URL to `the-paperless-project`.

File: docs/setup.rst [hunk 2]
```diff
@@ -484,3 +489,45 @@ If you're using Docker, you can set a restart-policy_ in the
...
+
+.. code:: bash
+
+    $ lxc launch ubuntu: paperless
+
+3. Run install script within container
+
+.. code:: bash
+
+    $ lxc exec paperless -- sh -c "wget https://raw.githubusercontent.com/danielquinn/paperless/master/docs/examples/lxc/lxc-install.sh && /bin/bash lxc-install.sh"
```
Comment (MasterofJOKers) [inline docs/setup.rst]: and here

File: docs/examples/lxc/lxc-install.sh [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+<Directory /home/ftpupload/>
+  UserOwner   paperless
+  GroupOwner  paperless
+</Directory>
+EOF
+systemctl restart proftpd
+
+
+#Get Paperless from git 
+su -c "cd /home/paperless ; git clone https://github.com/danielquinn/paperless" paperless
```
Comment (MasterofJOKers) [inline docs/examples/lxc/lxc-install.sh]: here, too.

File: docs/examples/lxc/lxc-install.sh [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+passwd ftpupload
+# Allow paperless group to access
+adduser paperless ftpupload
+chmod g+w /home/ftpupload 
+
+# Get apt up to date
+apt-get update
+
+# Needed for plain Paperless
+apt-get -y install unpaper gnupg libpoppler-cpp-dev python3-pyocr tesseract-ocr imagemagick optipng
```
Comment (MasterofJOKers) [inline docs/examples/lxc/lxc-install.sh]: What about `git`, is that already installed per assumption?

File: docs/examples/lxc/lxc-install.sh [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+cd /home/paperless/paperless
+pip3 install -r requirements.txt
+
+# Take paperless.conf.example and set consumuption dir (ftp dir)
+sed  -e '/PAPERLESS_CONSUMPTION_DIR=/s/=.*/=\"\/home\/ftpupload\/\"/' \
+     /home/paperless/paperless/paperless.conf.example  >/etc/paperless.conf
+
+# Update /etc/paperless.conf with PAPERLESS_SECRET_KEY
+SECRET=$(strings /dev/urandom | grep -o '[[:alnum:]]' | head -n 30 | tr -d '\n'; echo)
+sed  -i "s/#PAPERLESS_SECRET_KEY.*/PAPERLESS_SECRET_KEY=$SECRET/g" /etc/paperless.conf
```
Comment (MasterofJOKers) [inline docs/examples/lxc/lxc-install.sh]: What do you need the `/g` for? I thought it's for multiple replacements, but there should only be one `PAPERLESS_SECRET_KEY`, right?

File: docs/examples/lxc/lxc-install.sh [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+# 167 static files copied to '/home/paperless/paperless/static'.
+su -c "cd /home/paperless/paperless/src/ ; ./manage.py collectstatic" paperless
+
+# Set-up apache
+cp /home/paperless/paperless/docs/examples/lxc/paperless.conf /etc/apache2/sites-available/
+a2dissite 000-default.conf
+a2ensite paperless.conf
+systemctl reload apache2
+
+sed -e "s/home\/paperless\/project\/virtualenv\/bin\/python/usr\/bin\/python3/" \
```
Comment (MasterofJOKers) [inline docs/examples/lxc/lxc-install.sh]: For readabilities sake, I'd prefer using `sed` with an alternative separator here, e.g. `sed -e 's!/home/paperless/project/virtualenv/bin/python!/usr/bin/python3!'`

File: docs/examples/lxc/lxc-install.sh [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+     /home/paperless/paperless/scripts/paperless-consumer.service \
+     >/etc/systemd/system/paperless-consumer.service
+
+sed  -i "s/\/home\/paperless\/project\/src\/manage.py/\/home\/paperless\/paperless\/src\/manage.py/" \
```
Comment (MasterofJOKers) [inline docs/examples/lxc/lxc-install.sh]: Same here. A different separator for sed might help readability.

**Comment by pitkley** [comment]
@bmsleight I‘ll close this PR since #543 has been merged. If there is anything you‘d like to add/update, you can obviously reopen this PR or create a new one. 👍 Thanks for the initial work on this!

**Pull request closed without merging**